### PR TITLE
Facet fix

### PR DESCRIPTION
**Pull request by goose-life**

### CODE REVIEW

File: indigo_app/forms.py [hunk 1]
```diff
@@ -558,17 +558,17 @@ def filter_queryset(self, queryset, exclude=None):
...
                 documents_qs |= Q(id__in=multiple_document_ids)
             if 'none' in documents_filter:
+                # either there are no documents at all
                 documents_qs |= Q(document__isnull=True)
-            if 'published' in documents_filter:
-                documents_qs |= Q(document__draft=False)
-            if 'draft' in documents_filter:
-                documents_qs |= Q(document__draft=True)
+                # or they're all deleted
+                all_deleted_document_ids = queryset.filter(document__deleted=True).annotate(Count('document')).filter(document__count=0).values_list('pk', flat=True)
```
Comment (goose-life) [inline indigo_app/forms.py:570]: I don't think this is right

**Comment by goose-life** [comment]
@actlikewill this _works_, but it feels super clunky to me. I hope at least the logic makes sense to you?

File: indigo_app/forms.py [hunk 2]
```diff
@@ -558,17 +558,19 @@ def filter_queryset(self, queryset, exclude=None):
...
                 documents_qs |= Q(document__isnull=True)
-            if 'published' in documents_filter:
-                documents_qs |= Q(document__draft=False)
-            if 'draft' in documents_filter:
-                documents_qs |= Q(document__draft=True)
+                # or they're all deleted
+                deleted_document_ids = queryset.filter(document__deleted=True).values_list('pk', flat=True)
+                undeleted_document_ids = queryset.filter(document__deleted=False).values_list('pk', flat=True)
+                all_deleted_document_ids = deleted_document_ids.exclude(id__in=undeleted_document_ids)
+                documents_qs |= Q(id__in=all_deleted_document_ids)
```
Comment (actlikewill) [inline indigo_app/forms.py:573]: should we include the draft=True here?

Comment (actlikewill) [reply indigo_app/forms.py:573]: oh nevermind :smile: I see that its under the `none` filter

**Review by actlikewill — APPROVED** [review]
This looks good and works as expected :+1:

**Pull request merged**